Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -1492,14 +1488,25 @@ - (_WKContentSecurityPolicyModeForExtension)_contentSecurityPolicyModeForExtensi
     return WebKit::toWKContentSecurityPolicyModeForExtension(_pageConfiguration->contentSecurityPolicyModeForExtension());
 }
 
+// Needed to maintain backwards compatability
```

Comment:
This should be replaced with a `FIXME` to remove the SPI. Please file bugs against all SPI clients too.